### PR TITLE
Update to release beta.16

### PR DESCRIPTION
**Pull request by JohnCashmore**
I've updated the project to release beta.16 and modified the tests, however it was a little rushed so not 100% they've been updated correctly.

### CODE REVIEW

**Comment by mgechev** [comment]
LGTM

File: src/client/app/+about/components/about.component.spec.ts [hunk 1]
```diff
@@ -2,25 +2,29 @@ import {
   TestComponentBuilder,
   describe,
   expect,
-  injectAsync,
-  it
-} from 'angular2/testing';
+  inject,
+  it,
+  AsyncTestCompleter
+} from 'angular2/testing_internal';
```
Comment (ludohenin) [inline src/client/app/+about/components/about.component.spec.ts]: It should use `testing` module as `testing_internal` is used to test the framework itself.

File: src/client/app/+about/components/about.component.spec.ts [hunk 2]
```diff
@@ -2,25 +2,29 @@ import {
...
 import {AboutComponent} from './about.component';
 
 export function main() {
   describe('About component', () => {
+
+
     it('should work',
-      injectAsync([TestComponentBuilder], (tcb: TestComponentBuilder) => {
-        return tcb.createAsync(TestComponent)
+      inject([TestComponentBuilder, AsyncTestCompleter], (tcb: TestComponentBuilder, async: AsyncTestCompleter) => {
```
Comment (ludohenin) [inline src/client/app/+about/components/about.component.spec.ts]: Why is `AsyncTestCompleter` required, as per the doc it should work without it ?

Comment (ludohenin) [inline src/client/app/+about/components/about.component.spec.ts]: It should actually be 
```
async(inject([TestComponentBuilder], (tcb: TestComponentBuilder) => {
  //...
}));
```

Comment (JohnCashmore) [inline src/client/app/+about/components/about.component.spec.ts]: I thought this too, but it fails with 

```
TypeError: undefined is not a constructor (evaluating 'new AsyncTestZoneSpec(finishCallback, failCallback, testName)') (line 98)
```

So i refered to the source to see how its implemented in the angular source

**Comment by JohnCashmore** [comment]
I've made some amends as per comments

**Pull request merged**